app/ledger/n3NeonLedger.js

```diff
@@ -233,6 +237,13 @@ export const signWithLedger = async (
   details: { network: number, witnessIndex: number },
   acct: number = 0,
 ): Promise<string> => {
+  const store = require('../store/configureStore').default
+
+  const id = showInfoNotification({
+    message: 'Please sign the transaction on your hardware device',
+    autoDismiss: 0,
+  })(store.dispatch)
```

Comment:
Have you tested this change?? We already dispatch this notification from other places in the application so adding it here is probably going to cause it to render twice